package.json

```diff
@@ -58,5 +58,9 @@
     "standard": "^17.0.0",
     "turndown": "^6.0.0"
   },
-  "dependencies": {}
+  "dependencies": {},
+  "babel": {
+    "presets": ["@babel/preset-env"]
+  },
+  "browserslist": "defaults, IE 11"
```

Comment:
Be good to move these up the file for easier grokability. I'd put them above jest and standard